util/logging.go

```diff
@@ -79,7 +79,7 @@ var (
 	LogHttpResponse bool = true
 
 	// List of tags to be excluded from logging
-	skipTags = []string{"SupportedVersions", "ovf:License"}
+	skipTags = []string{"ovf:License"}
```

Comment:
the SupportedVersions call in recent releases has become short and bearable, while up to 9.5 it was 600+ lines.